### PR TITLE
Pt 52

### PR DESCRIPTION
**Pull request by MartinHwang**
I completed popup to go to writing chapter in Content Creation Page.

### CODE REVIEW

File: src/pages/book/publish.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+              key={scream.screamId}
+              scream={scream}
+              div_name={scream.createdAt}
+            />
+          );
+        else
+          return <BookCard key={scream.screamId} scream={scream} openDialog />;
+      })
+    );
+    let confirmDialog = (
```
Comment (Harukaichii) [inline src/pages/book/publish.js:122]: you don't have an image on the side like the ticket described, Is that ok?

**Review by Harukaichii — CHANGES_REQUESTED** [review]
Functionality works as described! again just some styling issues you should ask someone about?

File: src/pages/book/publish.js (hunk 1 above)
Comment (MartinHwang) [reply src/pages/book/publish.js:122]: > Functionality works as described! again just some styling issues you should ask someone about?

I will talk about that with daughter who is taking art in her school.

Comment (MartinHwang) [reply src/pages/book/publish.js:122]: > you don't have an image on the side like the ticket described, Is that ok?

Can you explain more please? I can not draw picture on my brain. so sorry for that

Comment (Harukaichii) [reply src/pages/book/publish.js:122]: https://proudtale.atlassian.net/browse/PT-52?jql=order%20by%20created%20DESC

Comment (Harukaichii) [reply src/pages/book/publish.js:122]: I'm talking about this

Comment (MartinHwang) [reply src/pages/book/publish.js:122]: > https://proudtale.atlassian.net/browse/PT-52?jql=order%20by%20created%20DESC

Now, I see. I will add proudtale image  on left side on the pop up. Thank you.

**Review by Harukaichii — APPROVED** [review]

**Pull request merged**